<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatelliteEditor</class>
 <widget class="QWidget" name="SatelliteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Спутник</string>
  </property>
  <layout class="QVBoxLayout" name="iSatelliteEditorVbox">
   <item>
    <layout class="QHBoxLayout" name="iSatHbox">
     <item>
      <widget class="QLabel" name="satLbl">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Спутник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="satCbox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iTypeHbox">
     <item>
      <widget class="QLabel" name="iTypeLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Орбита</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="iReceiverGbox">
     <property name="title">
      <string>Параметры приёма</string>
     </property>
     <layout class="QVBoxLayout" name="iReceiverVbox">
      <item>
       <widget class="QTableWidget" name="receiverTbl">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Тип</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Частота</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Усиление</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Частота дискретизации</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iControlsHbox">
     <item>
      <widget class="QPushButton" name="addReceiverBtn">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeReceiverBtn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeCbox</tabstop>
  <tabstop>receiverTbl</tabstop>
  <tabstop>addReceiverBtn</tabstop>
  <tabstop>removeReceiverBtn</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
